src/components/sections/image-split/ImageSplit.tsx

```diff
@@ -0,0 +1,61 @@
+import { PortableText, PortableTextReactComponents } from "@portabletext/react";
+
+import { SanityImage } from "src/components/image/SanityImage";
+import LinkButton from "src/components/linkButton/LinkButton";
+import Text from "src/components/text/Text";
+import { ImageSplitSection } from "studio/lib/interfaces/pages";
+import { ImageAlignment } from "studio/schemas/fields/media";
+
+import styles from "./image-split.module.css";
+
+interface ImageSplitProps {
+  section: ImageSplitSection;
+}
+
+const myPortableTextComponents: Partial<PortableTextReactComponents> = {
+  block: ({ children }) => <Text type="bodyNormal">{children}</Text>,
+};
+
+const ImageSplitComponent = ({ section }: ImageSplitProps) => {
+  const hasImage = section.imageExtended;
+  const alignment = section.imageExtended?.imageAlignment;
+  const showImageToLeft = hasImage && alignment == ImageAlignment.Left;
+  const showImageToRight = hasImage && alignment == ImageAlignment.Right;
+
+  return (
+    <article className={styles.imageSplit}>
+      {showImageToLeft && (
```

Comment:
Could do this css only with order to avoid having image be fore the title in the actual DOM, but would require not to use grid and I used grid to more robustly get it as 50/50 size. 🤔 Might be lazy